### PR TITLE
Lazarus injectors now grab the ghosts of revivees

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
add: A sentient animal, revived with a lazarus injector, will have their ghost grabbed out of the afterlife.
/:cl:

### CODE REVIEW

**Comment by GunHog** [comment]
It should not be forced, imo. Why not just give it an alert?

**Comment by coiax** [comment]
If you didn't want to have your ghost forced out of the afterlife, then you shouldn't have died.

File: code/modules/mining/equipment.dm [hunk 1]
```diff
@@ -304,6 +304,9 @@
 						log_game("[user] has revived hostile mob [target] with a malfunctioning lazarus injector")
 					else
 						H.attack_same = 0
+				M.grab_ghost(force=TRUE)
+				M << "<span class='notice'>You rise with a start, \
```
Comment (RemieRichards) [inline code/modules/mining/equipment.dm]: stop with thiiiis.

Comment (ChangelingRain) [inline code/modules/mining/equipment.dm]: seriously quit it with the super short line lengths

**Comment by optimumtact** [comment]
Make it an alert, people who have left their body may not wish to re-enter it.

**Pull request closed without merging**